Makefile

```diff
@@ -0,0 +1,24 @@
+ifndef VERBOSE
+.SILENT:
+endif
+
+nonprod: build_nonprod clean
+	echo "Build complete"
```

Comment:
You can set it to a different [recipe prefix](https://www.gnu.org/software/make/manual/html_node/Special-Variables.html) but the convention is to use tabs.